tests/installation/partitioning_finish.pm

```diff
@@ -18,7 +18,9 @@ use testapi;
 sub run {
     wait_still_screen();
     send_key $cmd{next};
+    wait_still_screen();
```

Comment:
that is a waste of 7 seconds at least in every graphical installation. Also, directly afterwards comes an assert_screen which should cover this. This IMHO is a no-go.